db_stress_tool/db_stress_test_base.cc

```diff
@@ -2491,5 +2571,72 @@ void StressTest::Reopen(ThreadState* thread) {
           clock_->TimeToString(now / 1000000).c_str(), num_times_reopened_);
   Open();
 }
+
+void StressTest::CheckAndSetOptionsForUserTimestamp() {
```

Comment:
Will it cause db_stress likely to fail? Should we skip the un-supported options in `db_crashtest.py`?

Reply:
This is already done.